### PR TITLE
publish command for ts

### PR DESCRIPTION
**Pull request by jckhang**
publish command for ts

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,14 +67,22 @@ lint-all: \
...
 	python3 -m pip install -U setuptools twine wheel
 	python3 setup.py sdist bdist_wheel
 	python3 -m twine upload dist/accern_xyme-$(VERSION)-py3-none-any.whl dist/accern_xyme-$(VERSION).tar.gz
 	git tag "v$(VERSION)"
 	git push origin "v$(VERSION)"
 	@echo "succesfully deployed $(VERSION)"
+
+publish-ts:
+	make git-check
+	yarn publish
```
Comment (JosuaKrause) [inline Makefile:87]: is there a way of retrieving the version that is in ts and check whether the tag exists?

Comment (JosuaKrause) [reply Makefile:87]: at least for non-rc?

Comment (jckhang) [reply Makefile:87]: yarn will error if you publishing one version already exist

Comment (JosuaKrause) [reply Makefile:87]: git tag

Comment (JosuaKrause) [reply Makefile:87]: the version published must match the git tag

**Comment by jckhang** [comment]
close by #59

**Pull request closed without merging**